core/test/debt-test.ts

```diff
@@ -0,0 +1,21 @@
+import { expect } from 'chai';
+import { causeDebt } from '../helpers/causeDebt';
+import getContract from '../src/contracts';
+import BigNumber from '../src/bignumber';
+import reset from './helpers/resetBlockchainFork';
+import { HARDHAT_PRIVATE_KEY, NETWORK } from '../helpers/constants';
+const HARDHAT_FORK_BLOCK_NUMBER = 14052140;
+
+const REMOTE_RPC_URL = process.env.REMOTE_RPC_URL || 'http://localhost:8545';
+
+describe('Debt Auction', () => {
+    beforeEach(async () => {
+        const provider = await reset(REMOTE_RPC_URL, HARDHAT_FORK_BLOCK_NUMBER, HARDHAT_PRIVATE_KEY, NETWORK);
+        await causeDebt(NETWORK, provider, new BigNumber(10), new BigNumber(1000), new BigNumber(1000));
```

Comment:
Do you actually need to pass/modify those variables in any circumstances?

Reply:
i don't understand the question.

All these are configurable so that one can enforce any state on the system freely to test various scenarios if needed.

Reply:
You can make _everything_ configurable, but the question is if it makes sense to provide possibility to change those values. The question was: would it actually be helpful to change those parameters to uncover any edge case? If not, why do we ever need to change those?

As a quick compromise, please move those values to the default values of the function, so it can be called without them
```suggestion
        await causeDebt(NETWORK, provider);
```